JS-Algorithms/challenges.js

```diff
@@ -70,7 +87,28 @@ The last 3 digits for the sum of powers from 1 to 10 is "317"
 ***** */
 
 const ownPower = (number, lastDigits) => {
-  // YOUR CODE HERE...
+  const modBase = Math.pow(10, lastDigits + 1);
```

Comment:
I like this approach to not overflow numbers and be precise in the calculations, you can also use a BigInt instead of it but good usage